### PR TITLE
Added Readme

### PR DESCRIPTION
**Pull request by AALUND13**
- Added Readme
- Rewrite Setup Symlink file

### CODE REVIEW

**Review by Tess-y — CHANGES_REQUESTED** [review]
Honestly, I disagree with the sim-link idea in the first place and would have rejected it if I had had the branch protection setup correctly before you had committed it to main. That being said. If you _really_ want to implement it that way, you must do so in a way that can function on all systems, not just windows. Otherwise I am considering reverting the change made to the csproj file.

**Comment by Scyye** [comment]
I dont see a need for a symlink file.
Is that not just making it so ya dont have to change a single variable in the csproj files, which takes ~10 seconds?

**Comment by Tess-y** [comment]
To my knowledge, that is the reason it is there. Which is why if it was a PR it would have been rejected.

**Comment by AALUND13** [comment]
Since I don't want to figure out how to recreate the same setup in a shell file, I may remove the need for it to be a symlink and rewrite the README, excluding any references to the symlink. The changes will be made within a few minutes to a few hours.

**Comment by AALUND13** [comment]
I can use Python to create a symlink that works on Windows, Linux/UNIX, and macOS. However, the end user must have Python installed on their system. What do you think about using python to create symlink?

**Comment by Scyye** [comment]
I dont see a reason for it.
Its fixing a non-issue, and having a weird requirement of python in order to develop a ROUNDS mod, is quite unnecessary

**Comment by Tess-y** [comment]
I agree, you are attempting to needlessly complicate what was as simple as changing a single variable in a single file.

**Comment by Tess-y** [comment]
if you can come up with a system that allows any user to do this without the need of installing additional external tools, then by all means go for it. but barring that I will be rejecting this pull request, and reverting the change that you made to the project file.

**Comment by AALUND13** [comment]
I am against using Python cause you have to install a extra software just to run one file

**Comment by Tess-y** [comment]
> I am against using Python cause you have to install a extra software just to run one file

you are quite literally the one who suggested using python the first place. what was the point of suggesting it if you are against it?

**Comment by AALUND13** [comment]
I just removed the need for a symlink and rewritten the readme cause to a complex of a non-issue just to solve one miner non-issue only save a few seconds at the end

**Review by Scyye — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-
-3. **Build the Project:**
-    - Open the project file.
-    - Press `Ctrl + B` to build.
+### 1. Update Steam Library Path in Project File:
+
+- Navigate to the `RoundsModdingTools` folder.
+- Open the file named `RoundsModdingTools.csproj`.
+- Locate the line resembling: `<SteamFolder>F:\SteamLibrary</SteamFolder>`.
+- Replace `F:\SteamLibrary` with the path to your Steam Library folder.
```
Comment (Tess-y) [inline README.md]: Change lines 11 and 12 to just say to set the SteamFolder variable, rather then explicitly referencing _my_ Steam Library.
then I'll accept the PR.

**Review by Tess-y — APPROVED** [review]

**Pull request merged**